sdk/nodejs/src/cloudClient.ts

```diff
@@ -32,7 +34,7 @@ export class DVCCloudClient {
         checkParamDefined('key', key)
         checkParamDefined('defaultValue', defaultValue)
 
-        return getVariable(requestUser, this.environmentKey, key)
+        return getVariable(requestUser, this.environmentKey, key, this.options.enableEdgeDB)
```

Comment:
Could options be null / undefined?

Reply:
`options` won't be... but enableEdgeDB might be undefined.. let me write a test for that usecase